conf/modules.config

```diff
@@ -1747,13 +1769,13 @@ process {
     }
 
     withName: FREEBAYES {
-        tag = { "${meta.reference}|${meta.sample_id}" }
+        tag = { params.genotyping_use_unmerged_libraries ? "${meta.reference}|${meta.sample_id}|${meta.library_id}" : "${meta.reference}|${meta.sample_id}" }
         ext.args = {[
-            "-p ${ref_meta.ploidy}",
+            ref_meta.ploidy ? "--ploidy ${ref_meta.ploidy}" : "--ploidy 2",
```

Comment:
I think we should use the fasta schema to provide a default value for this instead of hard-coding default values everywhere.